mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -188,8 +188,6 @@ private boolean isDataSaveNecessary(EditText input) {
     public void onUserDetailsFocusChanged(EditText input, boolean isFocused) {
         if (!isDataSaveNecessary((input))) {
```

Comment:
Combine the nested ifs in this and next line
```suggestion
        if (!isDataSaveNecessary((input)) && isFocused) {
```